content/documentation/pages/4-batch-developer-guides/4-continuous-deployment/0-index.md

```diff
@@ -0,0 +1,16 @@
+---
+path: 'batch-developer-guides/continuous-deployment/'
+title: 'Continuous Deployment'
+description: 'Continuous Deployment for task applications'
+summary: true
+---
+
+# Continuous Deployment
+
+Spring Cloud Data Flow provides native support for continuous deployment of task applications.
+
+The application registry in Spring Cloud Data Flow lets you register multiple versions for the same task application.
+
+When launch the task applications, you can choose which version of the task application you want to use for the launch. You can also re-use and update the deployment properties used from the previous launch of the same task.
```

Comment:
```suggestion
When launching the task applications, you can choose which version of the task application you want to use for the launch. You can also re-use and update the deployment properties used from the previous launch of the same task.
```

Reply:
So sorry about getting to this so late.   `When reading the line  You can also re-use and update the deployment properties used from the previous launch of the same task.`    It should read something like:
`Data Flow re-uses the deployment properties used from the previous launch of the same task.  You can update these properties for each launch.`   If you agree I can updated on merge.